docs/modules/ROOT/pages/ai-services.adoc

```diff
@@ -113,6 +113,14 @@ TriagedReview triage(String review);
 
 In this instance, Quarkus automatically creates an instance of `TriagedReview` from the LLM's JSON response.
 
+To enhance the flexibility of prompt creation, the `{response_schema}` placeholder can be used within the prompt. This placeholder is dynamically replaced with the defined schema of the method's return object.
```

Comment:
```suggestion
To enhance the flexibility of prompt creation, the `+{response_schema}+` placeholder can be used within the prompt. This placeholder is dynamically replaced with the defined schema of the method's return object.
```

This is to avoid Antora trying to resolve this as an Antora attribute named `response_schema` - I can see this in the build log:
```
[WARNING] [io.quarkiverse.antora.deployment.NativeImageBuildRunner] /antora/docs/modules/ROOT/pages/ai-services.adoc:0: skipping reference to missing attribute: response_schema
```

Reply:
It seems that one solution to avoid the warning is to use `\{response_schema\}`.